oneanddone/tasks/templates/tasks/task_list_filters.html

```diff
@@ -10,6 +10,10 @@
           {{ filter.form['search'] }}
         </p>
         <p>
+          {{ filter.form['difficulty'].label_tag() }}
+          {{ filter.form['difficulty'] }}
```

Comment:
This doesn't end up looking very good on the form, because the labels and boxes don't all appear on one line. It's fine for `execution_time` because those labels are small, but this doesn't look very nice. I suggest outputting these similarly to how `team` and `project` are done.

![screenshot 2015-01-19 16 22 50](https://cloud.githubusercontent.com/assets/142755/5808080/7153c750-9ff7-11e4-99ba-b7c6226bca2a.png)